vault/datadog_checks/vault/check.py

```diff
@@ -85,13 +85,18 @@ def check_leader_v1(self, submission_queue, dynamic_tags):
             return
 
         is_leader = is_affirmative(leader_data.get('is_self'))
-        dynamic_tags.append(f'is_leader:{str(is_leader).lower()}')
 
         submission_queue.append(lambda tags: self.gauge('is_leader', int(is_leader), tags=tags))
 
         current_leader = Leader(leader_data.get('leader_address'), leader_data.get('leader_cluster_address'))
         has_leader = any(current_leader)  # At least one address is set
 
+        if self._previous_leader is not None:
+            has_leader_tags = list(self._tags)
+            has_leader_tags.extend(dynamic_tags)
+            has_leader_tags.append(f'current_leader:{current_leader}')
+            submission_queue.append(lambda tags: self.gauge('has_leader', int(has_leader), tags=has_leader_tags))
```

Comment:
```suggestion
            submission_queue.append(lambda tags: self.gauge('vault.has_leader', int(has_leader), tags=has_leader_tags))
```

Reply:
@steveny91 it is working as expected the way it is in the PR, if you see line 89 on the same file you'll notice that the existing is_leader gauge doesn't need the vault. prefix either and it's being sumitted the same way :)